Web/Edubase.Web.UI/Areas/Establishments/Views/AcademyOpenings/Index.cshtml

```diff
@@ -150,14 +150,14 @@
                         </tr>
                         </thead>
                         <tbody v-if="pages.length > 0" class="govuk-table__body">
-                        <tr v-for="(entry, index) in page" class="govuk-table__row">
-                            <td data-label="Opening date" class="govuk-table__cell cell-openingdate"> {{entry.displayDate}} </td>
-                            <td data-label="URN" class="govuk-table__cell"><a v-bind:href="detailUrl(entry.urn)">{{entry.urn}}</a></td>
-                            <td data-label="Establishment name" class="govuk-table__cell">{{entry.name}}</td>
-                            <td data-label="Establishment type" class="govuk-table__cell">{{entry.establishmentType}}</td>
-                            <td data-label="Predecessor URN" class="govuk-table__cell"><a v-bind:href="detailUrl(entry.predecessorUrn)" v-if="entry.predecessorUrn">{{entry.predecessorUrn}}</a></td>
-                            <td data-label="Predecessor name" class="govuk-table__cell">{{entry.predecessorName}}</td>
-                            <td class="cell-edit govuk-table__cell"><a href="#" v-on:click.prevent="editEstab(entry.urn)" aria-label="Edit opening details">Edit</a></td>
+                        <tr v-for="(entry, index) in page" class="govuk-table__row" id="table-row">
```

Comment:
I interpret this to mean that every row be assigned the same `id` value, which is likely unwanted/invalid HTML output.

I'm a little rusty so apologies if the syntax is off/needs tweaking, but perhaps something like this might work?

```vue
<tr v-for="(entry, index) in page" class="govuk-table__row" v-bind:id="`academy-opening-table-row-${index}`">
```

Or, perhaps:

```vue
<tr v-for="(entry, index) in page" class="govuk-table__row" v-bind:id="`academy-opening-table-row-${entry.urn}`">
```

Reply:
Good spot @RogerHowellDfE. Change made.